### PR TITLE
[DM-29112] Turn on cachemachine / nublado2

### PR DESCRIPTION
**Pull request by cbanek**
This is for idf related environments, as requested.

### CODE REVIEW

File: services/nublado2/values-idfint.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        tag: "master"
+
+    singleuser:
+      storage:
+        type: none
+        extraVolumes:
+          - name: datasets
+            nfs:
+              path: /share1/datasets
+              server: 10.87.86.26
```
Comment (athornton) [inline services/nublado2/values-idfint.yaml:18]: Server is 10.22.240.130 in int

File: services/nublado2/values-idfprod.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        tag: "master"
+
+    singleuser:
+      storage:
+        type: none
+        extraVolumes:
+          - name: datasets
+            nfs:
+              path: /share1/datasets
+              server: 10.87.86.26
```
Comment (athornton) [inline services/nublado2/values-idfprod.yaml:18]: Server is 10.13.105.122 in prod

**Review by athornton — CHANGES_REQUESTED** [review]
Unlike NCSA, we have different filestores in our three environments at IDF.  See comments for their IP addresses.

**Comment by cbanek** [comment]
Thanks, fixed.

**Review by athornton — APPROVED** [review]
LGTM

**Pull request merged**